ci_framework/roles/switch_config/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+# Amount of retries to supported tasks
+cifmw_switch_config_task_retries: 3
```

Comment:
Care to list those variables in the README.md?
All roles have that, in case you want to have a look regarding the layout.

Reply:
added